CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Yarn
+
+Yarn is in a slightly precarious position, due to the odd nature of MCP. MCP is all-rights-reserved, yet visible to anyone
```

Comment:
Drop the MCP stuff from CONTRIBUTING.md, IMO. I think the big thick warning in README.md is enough.